containers/src/Kokkos_DynRankView.hpp

```diff
@@ -1946,9 +1926,9 @@ inline auto create_mirror(const DynRankView<T, P...>& src,
       arg_prop, std::string(src.label()).append("_mirror"));
 
   if constexpr (Impl::ViewCtorProp<ViewCtorArgs...>::has_memory_space) {
-    using dst_type = typename Impl::MirrorDRVType<
+    using dst_type = typename Impl::MirrorDRViewType<
```

Comment:
This may conflict with Christian's ongoing work (on my cell so don't have the reference handy)
Also we are trying to break that pattern of containers using core implementation details

Reply:
> This may conflict with Christian's ongoing work (on my cell so don't have the reference handy)

https://github.com/kokkos/kokkos/pull/7342.

Reply:
Test passed after merging Christian's refactoring work, so I consider this is good.